components/eamxx/cime_config/testdefs/testmods_dirs/scream/output/README

```diff
@@ -0,0 +1,9 @@
+The testmods in this folder contain different configuration
+of output to request to eamxx. All the presets create a single
+output file, all with a different name, which means one can
+use 2+ presets at the same time.
+
+
+Each preset does basically two things:
+ 1. create a yaml file in the case folder
+ 2. add the yaml file to the yaml_output_files xml setting of eamxx
```

Comment:
It looks like we have two macros, `YAML_GEN` and `YAML_EDIT`.  Could this README have a few comments briefly describing the use of each.  Namely that `YAML_GEN` is always needed, and `YAML_EDIT` can be used to add more changes.

Side question I had while reading test mods.  If we wanted a test like the current default output, which has output on both the Dynamics and Physics grids in a single output stream, then I'm guessing the way to set that up would be to use YAML_GEN to add the Physics output and then use YAML_EDIT to add Dynamics output?  Is the tool sophisticated enough to add a subfield to the main `Fields` key in the yaml file?